bundles/edu.kit.kastel.sdq.eclipse.common.view/src/edu/kit/kastel/eclipse/common/view/preferences/AdvancedPreferences.java

```diff
@@ -40,8 +41,14 @@ public void createFieldEditors() {
 		var overrideDefaultPreferences = new BooleanFieldEditor(PreferenceConstants.OVERRIDE_DEFAULT_PREFERENCES, "Tweak Eclipse Preferences on startup",
 				parent);
 
-		this.addField(absoluteConfigPath);
+		var columnsForGradingButtons = new IntegerFieldEditor(PreferenceConstants.GRADING_BUTTONS_IN_COLUMN, "Amount of Grading Buttons in one column", parent);
```

Comment:
Wouldn't this technically be "in one row"? :D

Reply:
```suggestion
		var columnsForGradingButtons = new IntegerFieldEditor(PreferenceConstants.GRADING_BUTTONS_IN_COLUMN, "Amount of Grading Buttons in one row", parent);
```